package.json

```diff
@@ -17,8 +17,11 @@
     "url": "https://github.com/maymonyu/First-Engine/issues"
   },
   "homepage": "https://github.com/maymonyu/First-Engine#readme",
-  "dependencies": {},
+  "dependencies": {
+    "csvtojson": "^2.0.10"
```

Comment:
we should also add @types/csvtojson to dev dependencies

Reply:
Done